src/main/resources/openapiv2.json

```diff
@@ -302,13 +274,31 @@
                 ],
```

Comment:
The subscription endpoint not only allow a list ot topics (described by `Topics` type) but also topic_pattern field. They are mutually exclusive, something that we should write in the description as well. @stanlyDoge @PaulRMellor

Reply:
Subscribes a consumer to one or more topics. You can describe the list of topics to which the consumer will subscribe in a list (of `Topics` type) or as a `topic_pattern` field.